adyenv6core/resources/adyenv6core-items.xml

```diff
@@ -202,6 +202,34 @@
                     <modifiers optional="false"/>
                     <defaultvalue>Boolean.FALSE</defaultvalue>
                 </attribute>
+                <attribute qualifier="adyenPOSEnabled" type="java.lang.boolean">
```

Comment:
Same in this file. Terminal API should not be anywhere.

Reply:
@rkewlani Can you review again ? thanks